File: CS-Examples/16_ImageAndShape/ResetShapeSize.cs
using System;
using System.Windows.Forms;
using Spire.Doc;
using Spire.Doc.Documents;
using Spire.Doc.Fields;

namespace ResetShapeSize
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            //Create a word document
			Document doc = new Document();

			//Load the file from disk
			doc.LoadFromFile(@"..\..\..\..\..\..\Data\Shapes.docx");

			//Get the first section 
			Section section = doc.Sections[0];

			//Get the first paragraph
			Paragraph para = section.Paragraphs[0];

			//Get the second shape
			ShapeObject shape = para.ChildObjects[1] as ShapeObject;

			//Reset the width and height of the shape
			shape.Width = 200;
			shape.Height = 200;

			//Save and launch document
			string output = "ResetShapeSize.docx";
			doc.SaveToFile(output, FileFormat.Docx);

			// Dispose the document
			doc.Dispose();
            Viewer(output);
        }
        private void Viewer(string fileName)
        {
            try
            {
                System.Diagnostics.Process.Start(fileName);
            }
            catch { }
        }
    }
}

File: VB-Examples/16_ImageAndShape/ResetShapeSize.vb
Imports Spire.Doc
Imports Spire.Doc.Documents
Imports Spire.Doc.Fields

Namespace ResetShapeSize
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			Dim input As String = "..\..\..\..\..\..\Data\Shapes.docx"

			'Create a word document
			Dim doc As New Document()

			'Load the file from disk
			doc.LoadFromFile(input)

			'Get the first section
			Dim section As Section = doc.Sections(0)

			'Get the first paragraph
			Dim para As Paragraph = section.Paragraphs(0)

			'Get the second shape
			Dim shape As ShapeObject = TryCast(para.ChildObjects(1), ShapeObject)

			'Reset the width and height of the shape
			shape.Width = 200
			shape.Height = 200

			'Save the document
			Dim output As String = "ResetShapeSize.docx"
			doc.SaveToFile(output, FileFormat.Docx)

			'Dispose the document
			doc.Dispose()
			
			Viewer(output)
		End Sub
		Private Sub Viewer(ByVal fileName As String)
			Try
				Process.Start(fileName)
			Catch
			End Try
		End Sub
	End Class
End Namespace
